src/QsCompiler/LanguageServer/TemporaryProject.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This file was accidentally committed and should be removed.

Reply:
Removed.